templates/etc/nginx/sites-available/fpm.conf.erb

```diff
@@ -41,7 +41,8 @@ server {
 
   location ~ ^(.+\.php)(.*)$ {
     include fastcgi_params;
-
+    #allows to handle 404 in case of a file not found in fpm
+    try_files $uri =404;
```

Comment:
Indeed, the `=404` means the default 404 page of nginx. So if the application can not handle it itself, nginx should do so.